### PR TITLE
Shorter tests/utils

### PR DESCRIPTION
**Pull request by dankenigsberg**
With its 4700 lines, tests/utils.go is too long for me to read and understand. This PR makes it ever so slightly simpler by make two of its functions private.

```release-note
NONE
```

### CODE REVIEW

File: tests/vmi_configuration_test.go [hunk 1]
```diff
@@ -65,6 +65,18 @@ import (
 
 type VMICreationFuncWithEFI func() *v1.VirtualMachineInstance
 
+func getNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.ResourceName) *k8sv1.Node {
```
[the remark this answers is not in the document]
Comment (dankenigsberg) [reply tests/vmi_configuration_test.go]: I don't mind.

**Review by EdDev — APPROVED** [review]
Thanks!

**Comment by EdDev** [comment]
> `tests/vmi_configuration_test.go:1077:26: cannot refer to unexported name tests.getNodeWithHugepages`
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/7522/pull-kubevirt-e2e-kind-1.22-sriov/1511620926649667584#1:build-log.txt%3A213

File: tests/vmi_configuration_test.go [hunk 2]
```diff
@@ -1074,7 +1074,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 					log.DefaultLogger().Object(hugepagesVmi).Infof("Fall back to use hugepages source file. Libvirt in the 1.16 provider version doesn't support memfd as memory backend")
 				}
 
-				nodeWithHugepages := tests.GetNodeWithHugepages(virtClient, hugepageType)
+				nodeWithHugepages := tests.getNodeWithHugepages(virtClient, hugepageType)
```
Comment (EdDev) [inline tests/vmi_configuration_test.go:1077]: need to drop the `tests.`

Comment (dankenigsberg) [reply tests/vmi_configuration_test.go]: yep, dropped.

**Review by orelmisan — APPROVED** [review]

**Comment by dankenigsberg** [comment]
Obsoleted by https://github.com/kubevirt/kubevirt/pull/7583

**Pull request closed without merging**